app/ensure_universal_wheels.py

```diff
@@ -71,9 +71,7 @@ def merge_wheels(url1, url2, dest_dir):
         platform_base_parts = platform.split("_")
         platform_base = "_".join(platform_base_parts[:3])
 
-        universal_wheel_path = (
-            dest_dir / f"{wheel_base.lower()}-{platform_base}_universal2.whl"
-        )
+        universal_wheel_path = dest_dir / f"{wheel_base.lower()}-{platform_base}_universal2.whl"
```

Comment:
This file was previously formatted with black. Did you increase the max line length? Oh I see there's a config for 120 chars. Did you pick that 120 chars setting? If so, what was your reasoning? Not per se against, but there's some advantage to be compatible with default black settings.

Reply:
Yep, I've been working with 120 for quite a long time, I find 88 to be particularly short compared to modern screens. Also, func/methods definition with hints become quite long, and 88 feels even shorter in that kind of situation.

Reply:
I am curious about @typemytype opinion on the matter : )

Reply:
I think 120 fine for the DrawBot codebase.